src/main/java/duke/main/Parser.java

```diff
@@ -0,0 +1,186 @@
+package duke.main;
+
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+import java.time.format.DateTimeParseException;
+
+import duke.command.AddCommand;
+import duke.command.Command;
+import duke.command.DeleteCommand;
+import duke.command.DoneCommand;
+import duke.command.ExitCommand;
+import duke.command.FindCommand;
+import duke.command.InvalidCommand;
+import duke.command.ListCommand;
+import duke.command.SaveCommand;
+import duke.task.Deadline;
+import duke.task.Event;
+import duke.task.ToDo;
+
+/**
+ * The Parser class holds static methods which are used to parse and evaluate user commands, ensuring the right command
+ * is executed.
+ */
+public class Parser {
+
+    /**
+     * Static method which takes in user commands and ensures the right command is executed in the Duke application.
+     *
+     * @param command The user command entered.
+     * @return Command This returns the Command class that should be executed.
+     * @throws DukeException On wrong syntax of command entered.
+     */
+    public static Command parse(String command) throws DukeException {
+        String first = command.toLowerCase().split(" ")[0];
+        switch (first) {
+        case "bye":
+            return new ExitCommand();
+        case "list":
+            return new ListCommand();
+        case "todo":
+            return parseToDo(command);
+        case "deadline":
+            return parseDeadline(command);
+        case "event":
+            return parseEvent(command);
+        case "delete":
+            return parseDelete(command);
+        case "done":
+            return parseDone(command);
+        case "save":
+            return new SaveCommand();
+        case "find":
+            return parseFind(command);
```

Comment:
Is there a reason why you did not add a `//Fallthrough` comment for the cases where you do not include a break statement?